src/08_comprehensions.py

```diff
@@ -11,22 +11,27 @@
 # Write a list comprehension to produce the array [1, 2, 3, 4, 5]
 
 y = []
-
+for x in range(6):
+    y.append(x)
 print (y)
 
```

Comment:
Try doing this with list comprehension which is just a concise way of writing one line code.